builtin/providers/aws/resource_aws_db_subnet_group.go

```diff
@@ -166,8 +165,9 @@ func resourceAwsDbSubnetGroupUpdate(d *schema.ResourceData, meta interface{}) er
 		}
 
 		_, err := conn.ModifyDBSubnetGroup(&rds.ModifyDBSubnetGroupInput{
-			DBSubnetGroupName: aws.String(d.Id()),
-			SubnetIds:         sIds,
+			DBSubnetGroupName:        aws.String(d.Id()),
+			DBSubnetGroupDescription: aws.String(d.Get("description").(string)),
```

Comment:
Fixed, added the HasChange on description as well. In the case where the subnet_ids do not change but description does change, the `GetChange` method is expected to still return the desired value (as it hasn't changed) for the update.